example/spec/parallel/lzardsBackup/LzardsBackupFromDistributionSpec.js

```diff
@@ -0,0 +1,153 @@
+        prefix,
+        {
+          files: [
+            {
+              bucket: 'protected',
+              regex: '^[^.]+\.jpg$',
+              lzards: { backup: true },
+              sampleFileName: 'asdf.jpg',
+            },
+            {
+              bucket: 'protected',
+              regex: '^[^.]+\.dat$',
+              sampleFileName: 'asdf.dat',
+            },
+          ],
+        }
+      );
+
+      const Payload = JSON.stringify({
+        cma: {
+          ReplaceConfig: {
+            Path: '$.payload',
+            TargetPath: '$.payload',
+          },
+          task_config: {
+            urlType: 'cloudfront',
+            cloudfrontEndpoint: 'http://d111111abcdef8.cloudfront.net/',
+            cumulus_message: {
+              outputs: [
+                {
+                  source: '{$.originalPayload}',
+                  destination: '{$.payload}',
+                },
+                {
+                  source: '{$.backupResults}',
+                  destination: '{$.meta.backupStatus}',
+                },
+              ],
+            },
+          },
+          event: {
+            cumulus_meta: {
+              system_bucket: config.bucket,
+            },
+            meta: {
+              buckets: config.buckets,
+              collection,
+              provider,
+              stack: config.stackName,
+            },
+            payload: {
+              granules: [
+                {
+                  granuleId: 'FakeGranule2',
+                  dataType: collection.name,
+                  version: collection.version,
+                  files: [
+                    {
+                      name: 'testGranule.jpg',
+                      filename: `s3://${ingestPath}/testGranule.jpg`,
+                      checksumType: 'md5',
+                      checksum: '5799f9560b232baf54337d334179caa0',
+                    },
+                    {
+                      name: 'testGranule.dat',
+                      filename: `s3://${ingestPath}/testGranule.dat`,
+                      checksumType: 'md5',
+                      checksum: '39a870a194a787550b6b5d1f49629236',
+                    },
+                  ],
+                },
+              ],
+            },
+          },
+        },
+      });
+
+      lzardsBackupOutput = await pTimeout(
+        lambda().invoke({ FunctionName, Payload }).promise(),
+        (functionConfig.Timeout + 10) * 1000
+      );
+    } catch (error) {
+      beforeAllFailed = true;
+      throw error;
+    }
+  });
+
+  it('succeeds', () => {
+    if (beforeAllFailed) fail('beforeAll() failed');
+    else {
+      expect(lzardsBackupOutput.FunctionError).toBe(undefined);
+    }
+  });
+
+  it('has the expected backup information', () => {
+    const backupStatus = JSON.parse(lzardsBackupOutput.Payload).meta.backupStatus;
+    expect(backupStatus[0].status).toBe('COMPLETED');
+    expect(backupStatus[0].statusCode).toBe(201);
```

Comment:
What do you mean? As far as I understand, the LZARDS backup task sends a request to LZARDS which will trigger an asynchronous process to back up the file on their side?

Reply:
Can't we check the output from the Lambda to see if the generated backup URLs match the expected values according to the bucket map?

Reply:
As I look closer at this test - all it seems to be verifying is that the request to LZARDS was successful? In the spirit of our retro conversation, do we need a full integration test for this? Is it necessary just because LZARDS is an external service which is mocked in unit testing?

I would somewhat buy that argument, except we already have `LzardsBackupSec.js` which proves that basic integration with LZARDS works. So any testing we need to do to show that our LZARDS task can handle different options seems like it could be handled at the unit level?

Reply:
Resolved thru conversation in Slack: New spec does test existing problems as LZARDS has certain input expectations that must be met in each case and of which only one expectation is tested in the existing spec; this spec tests the other.